src/utilities.js

```diff
@@ -120,9 +120,16 @@ export function parseResults(results) {
  * @return {Array}     Returns an array of comparisonItems
  */
 export function formatComparisonItems(obj) {
+  const isMultiRegion = obj.geo && Array.isArray(obj.geo);
+  const isMultiKeyword = () => Array.isArray(obj.keyword);
```

Comment:
Can we make this just a constant rather than a function since it doesn't actually take any parameters?
`const isMultiKeyword = Array.isArray(obj.keyword)`

Reply:
Good call! Addressed in https://github.com/pat310/google-trends-api/pull/116/commits/d9090f4e336226b5260d1cd12ea092af9c862bf8